src/ContactComponent.php

```diff
@@ -71,7 +74,7 @@ function _webform_csv_data_civicrm_contact($component, $export_options, $value)
   /**
    * Returns a list of contacts based on component settings.
    *
-   * @param \Drupal\webform\WebformInterface $node
+   * @param WebformInterface $node
```

Comment:
It's interesting that rector would suggest this since phpcs with the drupal rules says `77 | ERROR   | [x] Data types in @param tags need to be fully namespaced`. None of the files pass linting anyway it just seems odd that rector doesn't use the drupal rules?

Reply:
Yes, seems like a known bug in rector - https://www.drupal.org/project/rector/issues/3269329. A fix was recently merged at https://github.com/palantirnet/drupal-rector/pull/209. I just re-executed the rector with this PR included and have updated the PR now. Thanks @demeritcowboy